src/test/java/tests/TestRegister.java

```diff
@@ -21,12 +22,24 @@ public void TC07() {
         Log.info("> Create fake credentials");
         String email = DataHelper.getRandomEmail();
         String password = DataHelper.getRandomPassword();
-        String pid = DataHelper.getRandomPIDNumber();
 
         Log.info("> Register a new account with valid credentials");
-        registerPage.register(email, password, password, pid);
+        registerPage.register(email, password, password, Constant.DEFAULT_PID);
 
         Log.info("> Verify the successful message.");
         Assert.assertEquals(registerPage.getSuccessMsg(), "You're here", "Success message does not match!");
```

Comment:
@InternDN21-DoHoaiLinh declare a variable for expected message

Reply:
I fixed it. Thank you.